dlrover/python/master/elastic_training/rdzv_manager.py

```diff
@@ -281,6 +282,14 @@ def join_rendezvous(
             self._node_rdzv_times[node_rank] = round(
                 self._lastcall_time - self._start_rdzv_ts, 2
             )
+            if self._error_monitor:
+                self._error_monitor.report_event(
+                    "info",
+                    node_id,
+                    "job rendezvous",
```

Comment:
All the key metrics need to be defined as constant. Better use capital letter to define the metrics.